apps/src/templates/studioHomepages/CoteacherInviteNotification.jsx

```diff
@@ -18,15 +18,34 @@ export const showCoteacherInviteNotification = coteacherInvite => {
   return !!coteacherInvite && DCDO.get('show-coteacher-ui', true);
 };
 
+export const showCoteacherForPlInviteNotification = coteacherInviteForPl => {
```

Comment:
Could we de-dup this function with the above and pass in either the pl or non-pl invite?

i.e.
```
 if (
    !(
      showCoteacherInviteNotification(coteacherInvite) ||
      showCoteacherInviteNotification(coteacherInviteForPl)
    )
  )
```

Reply:
I think I addressed this one by creating just one function that takes `invite` as an argument... but please let me know if this addresses something different.